app/src/main/java/com/android/partagix/ui/components/ItemUi.kt

```diff
@@ -44,264 +48,469 @@ import androidx.compose.ui.text.style.TextOverflow
+        }
+        Spacer(modifier = Modifier.height(2.dp))
+        Text("Other contact: ", fontSize = smallerFontSize, lineHeight = 1.3.em)
+        if (u.favorite != listOf(false, false, false)) {
+          val email = if (u.favorite?.get(0) == false && u.email != "") "Email : ${u.email}" else ""
+          val phone =
+              if (u.favorite?.get(1) == false && u.phoneNumber != "") "Phone : ${user.phoneNumber}"
+              else ""
+          val telegram =
+              if (u.favorite?.get(2) == false && u.telegram != "") "Telegram : ${user.telegram}"
+              else ""
+          if (email.isEmpty() && phone.isEmpty() && telegram.isEmpty())
+              Text("No other contact", fontSize = 11.sp, lineHeight = 1.3.em)
+          else {
+            listOf(email, phone, telegram).filter { it.isNotEmpty() }
+            listOf(email, phone, telegram)
+                .filter { it.isNotEmpty() }
+                .forEach { contact -> ClickableText(contact) }
           }
+        } else {
+          Text("No other contact", fontSize = smallerFontSize)
         }
-  } else {
+        Spacer(modifier = Modifier.height(8.dp))
+
+        Row(
+            horizontalArrangement = Arrangement.Absolute.Right,
+            modifier = Modifier.fillMaxWidth().testTag("manageLoanScreenItemCardExpanded")) {
+              if (!isOutgoing) {
+                Button(
+                    onClick = { manageLoanViewModel.acceptLoan(loan, index) },
+                    content = {
+                      Icon(
+                          Icons.Default.Check, contentDescription = "validate", modifier = Modifier)
+                      Spacer(Modifier.width(3.dp))
+                      Text(text = "Validate")
+                    },
+                    modifier = Modifier.requiredWidth(100.dp).requiredHeight(32.dp),
+                    contentPadding = PaddingValues(3.dp, 0.dp, 7.dp, 0.dp))
+              }
+              Spacer(modifier = Modifier.width(6.dp))
+              Button(
+                  onClick = { manageLoanViewModel.declineLoan(loan, index) },
+                  content = {
+                    Icon(Icons.Default.Close, contentDescription = "cancel", modifier = Modifier)
+                    Spacer(Modifier.width(2.dp))
+                    Text(text = "Reject")
+                  },
+                  colors =
+                      ButtonColors(
+                          containerColor = MaterialTheme.colorScheme.error,
+                          contentColor = MaterialTheme.colorScheme.onError,
+                          disabledContainerColor = MaterialTheme.colorScheme.error,
+                          disabledContentColor = MaterialTheme.colorScheme.onError),
+                  modifier = Modifier.requiredWidth(100.dp).requiredHeight(32.dp),
+                  contentPadding = PaddingValues(3.dp, 0.dp, 7.dp, 0.dp))
+            }
+      }
+    }
+  }
+  /*} else {
```

Comment:
Maybe this commented code will be gone with the merge with the main, but make sure it won't be visible at the end